### PR TITLE
Add Datamuse API

### PR DESCRIPTION
**Pull request by andrealps**
Add Datamuse API to Open Source Projects

- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -759,6 +759,7 @@ API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [Countly](https://api.count.ly/reference) | Countly web analytics | No | No | Unknown |
 | [Creative Commons Catalog](https://api.creativecommons.engineering/) | Search among openly licensed and public domain works | `OAuth` | Yes | Yes |
+| [Datamuse](http://www.datamuse.com/api/) | Word-finding query engine | No | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:762]: If this API supports HTTPS, use this link here.

```suggestion
| [Datamuse](https://www.datamuse.com/api/) | Word-finding query engine | No | Yes | Unknown |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @andrealps 😄 

Can you resolve this review, please?

**Comment by andrealps** [comment]
Sorry, now it's fixed 😄

**Review by matheusfelipeog — APPROVED** [review]
A big THANKS for your update @andrealps 😄

**Pull request merged**